### PR TITLE
Fix tabnabbing vulnerability in Snow theme

### LINKED ISSUE #2438: Tabnabbing vulnerability in snow theme
**Steps for Reproduction**

1. Visit https://quilljs.com/standalone/snow/
2. Create a link to some url
3. Click the link to get the link preview
4. Inspect the link element

**Expected behavior**:
The link has the target attribute set to _blank but has no rel property. This means that documents containing untrusted links make the page they are embedded in susceptible to tabnabbing https://www.owasp.org/index.php/Reverse_Tabnabbing.

It would be expected that the rel property be set to noopener (possibly also norefferer and nofollow)

**Actual behavior**:
No rel property is set

**Platforms**:
All
Include browser, operating system and respective versions

**Version**:
All
Run `Quill.version` to find out

### PR DESCRIPTION
**Pull request by jonathanlloyd**
Fixes #2438

The link has the target attribute set to _blank but has no rel property. This means that documents containing untrusted links make the page they are embedded in susceptible to tabnabbing https://www.owasp.org/index.php/Reverse_Tabnabbing.
This PR sets the rel property to noopener (also norefferer and nofollow)

### CODE REVIEW

File: themes/snow.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ class SnowTooltip extends BaseTooltip {
   }
 }
 SnowTooltip.TEMPLATE = [
-  '<a class="ql-preview" target="_blank" href="about:blank"></a>',
+  '<a class="ql-preview" rel="noopener noreferrer nofollow" target="_blank" href="about:blank"></a>',
```
Comment (jhchen) [inline themes/snow.js]: `nofollow` does not seem relevant to tabnabbing?

Comment (jonathanlloyd) [reply themes/snow.js]: Fair, `noopener` is sufficient to mitigate tabnabbing. I've left in noreferrer for privacy reasons but can remove that too if you think it's too much.

Comment (jhchen) [reply themes/snow.js]: That's fine these two are what React's linter suggests as well

**Comment by jhchen** [comment]
Thanks for the PR!

**Pull request merged**